conf/messages

```diff
@@ -77,4 +77,7 @@ AddressSearchResult.link    = Select property
 confirmAddress.pageTitle = Confirm Address
 confirmAddress.caption   = Register for the business rates valuation service
 confirmAddress.heading   = Do you want to use this address?
-confirmAddress.radio.unselected.error = This field is required
\ No newline at end of file
+confirmAddress.radio.unselected.error = This field is required
+
+featureNotAvailable.title = This part of the online service is not available yet
+featureNotAvailable.idTrue = The Journey ID is {0}
```

Comment:
Add another message if a journey ID is not supplied e.g. featureNotAvailable.idFalse = No journey ID supplied